forgerock-openbanking-model/src/main/java/com/forgerock/openbanking/model/SoftwareStatement.java

```diff
@@ -52,29 +56,25 @@
     public Date updated;
     private String applicationId;
 
-
     /**
      * The software ID is the same as the ForgeRock Application Id. It is included here to make logic simpler for
      * TPPS wishing to support message signing with both ForgeRock certificates and OB issued certificates. OB issued
      * Software Statements contain software_id and org_id fields and specify that these values should be used in the
      * iss field of a jws header (message signing header). Having both the software_id and org_id fields in the
      * ForgeRock software statement will make TPP and our own test logic easier when performing message signing.
      */
-    private String software_id;
+    @JsonProperty("software_id")
+    private String softwareId;
```

Comment:
What will the generated lombok getter be for this?

Reply:
`getSoftwareId()`, but it will appear as `software_id` in JSON.